### PR TITLE
Tree fixes

### PR DESCRIPTION
**Pull request by diegopinate**
1. Added invert functionality to categories tree to be consistent with models tree
2. Added forced 100% height on content wrapper to avoid issues when using iframes to host design review in Teams.

### CODE REVIEW

File: packages/tree-widget/src/components/trees/CategoriesTree.tsx [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+  CategoryTree,
+  toggleAllCategories,
+  getCategories,
+} from "@bentley/ui-framework";
 import { IconButton } from "../IconButton";
 import { SearchBar } from "../search-bar/SearchBar";
 import { useTreeFilteringState } from "../TreeFilteringState";
 import "./CategoriesTree.scss";
 import { TreeWidget } from "../../TreeWidget";
+import { CategoryVisibilityHandler } from "@bentley/ui-framework/lib/ui-framework/imodel-components/category-tree/CategoryVisibilityHandler";
```
Comment (aruniverse) [inline packages/tree-widget/src/components/trees/CategoriesTree.tsx:21]: Could you import `CategoryVisibilityHandler` from the root level barrel file in ui-framework, so just add it to the existing imports from ui-framework

File: packages/tree-widget/src/components/trees/CategoriesTree.tsx [hunk 2]
```diff
@@ -53,6 +58,40 @@ export function CategoriesTreeComponent(props: CategoriesTreeComponentProps) {
     );
   }, [props.iModel, filteredProvider]);
 
+  const invert = React.useCallback(async () => {
+    const activeView = IModelApp.viewManager.getFirstOpenView();
+    const ids = await getCategories(props.iModel, activeView, filteredProvider);
+    if (!activeView) {
```
Comment (aruniverse) [inline packages/tree-widget/src/components/trees/CategoriesTree.tsx:64]: wouldn't you want to check if `activeView` is defined prior to calling `getCategories`?

Comment (diegopinate) [reply packages/tree-widget/src/components/trees/CategoriesTree.tsx:64]: That call takes an undefined viewport, but yes, in this case it makes sense to do it after. I'll fix it. Thx

File: packages/tree-widget/src/components/trees/CategoriesTree.tsx (hunk 1 above)
Comment (diegopinate) [reply packages/tree-widget/src/components/trees/CategoriesTree.tsx:21]: Nope, is internal atm, would need to update core first.

File: packages/tree-widget/src/components/trees/CategoriesTree.tsx (hunk 2 above)
Comment (diegopinate) [reply packages/tree-widget/src/components/trees/CategoriesTree.tsx:63]: Fixed.

File: packages/tree-widget/src/components/trees/CategoriesTree.tsx (hunk 1 above)
Comment (aruniverse) [reply packages/tree-widget/src/components/trees/CategoriesTree.tsx:21]: damn, thats unfortunate.

**Pull request merged**